src/resolve/resolved.c

```diff
@@ -93,6 +93,9 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return log_error_errno(r, "Event loop failed: %m");
 
+        /* send queries on shutdown to other servers */
+        manager_symlink_stub_to_uplink_resolv_conf();
```

Comment:
hmm, i'd prefer if we didn't do this blindly, but only if the target symlinks actually exists. And that we remove PRIVATE_STUB_RESOLV_CONF otherwise. (which will leave the symlink /etc/resolve.conf typically dangling, but that's fine).

Reply:
I quite do not get that. Dangling symlink leading nowhere is okay, when it breaks resolution. But /etc/resolv.conf missing is not okay and first service willing to take ownership of it is free to take it.

Reply:
well, if we leave it danling, or if we let it point to a stub that isn't running is both broken. I just think that a ENOENT when reading /etc/resolv.conf is nicer than the stub simply not responding as an error to handle.